docs/integrations/delta.md

```diff
@@ -70,6 +80,43 @@ The new action tries to contact lakeFS from servers on Databricks’ own AWS acc
 So, if you must use multi-cluster writes, you’ll have to allow access from Databricks’ AWS account to lakeFS. 
 If you are trying to achieve that, please reach out on Slack and the community will try to assist.
 
+### Customizing the Delta Lake diff plugin location
```

Comment:
Not sure this belongs under "Best Practices", but regardless, I think it could be condensed significantly and just moved to a note at the end of the `Installing the Delta Lake diff plugin` section, e.g. 

```
You can customize the location of the Delta Lake diff plugin by changing the `plugins.properties.diff.path` configuration in the [`.lakefs.yaml`](../reference/configuration) file. 
```

I don't see the point in documenting `diff.delta.plugin` unless there's a reason why users would want/need to change it (and if there is you probably should expand on it in the docs as it's not clear to me)

Reply:
True, this was accidentally placed under "Best Practices".
I'll also insert the changes you describe.
Thanks.